docs/installation/application.rst

```diff
@@ -0,0 +1,196 @@
+* **interactive**: If enabled, it will start an interactive mode that will let you select from existing portal services. This option is set to `False` by default. 
+
+
+* **persistent** : List of persistent store settings in the app and the service to link to each.
+* **dataset** : List of dataset settings in the app and the service to link to each.
+* **spatial** : List of spatial persistent store settings in the app and the service to link to each.
+* **wps** : List of Web Processing service settings in the app and the service to link to each. 
+
+Settings in each of the service sections above will need to be listed in the following format::
+
+	<app_service_setting_name> : <service_name or id>
+
+In the above example, ``catalog_db`` is the name of the service in your :file:`app.py` and ``hydroexplorer-persistent`` is the name of the service on the portal. 
+
+.. tip::
+
+    Run ``tethys gen services`` to create a blank template of this file. By default the file will be saved to the current working directory.
+
+portal.yml 
+------------
+
+The file is designed to be maintained by the server administrator who can provide incoming apps with default services. 
+
+.. code-block:: python
+
+	# Portal Level Config File
+
+	version: 1.0
+	name: Tethys Main Portal
+	apps:
+	 hydroexplorer:
+	  services:
+	   persistent:
+	    catalog_db: test
+	   spatial:
+	   dataset:
+	   wps:
+
+
+**portal.yml Options:**
+
+* **version**: Indicated the version of the :file:`portal.yml` file. Current default : 1.0
+* **name**: Name of the portal
+
+* **apps/<app-name>/services/persistent** : List of persistent store settings in the app and the service to link to each.
+* **apps/<app-name>/services/dataset** : List of dataset settings in the app and the service to link to each.
+* **apps/<app-name>/services/spatial** : List of spatial persistent store settings in the app and the service to link to each.
+* **apps/<app-name>/services/wps** : List of Web Processing service settings in the app and the service to link to each. 
+
+Settings in each of the service sections above will need to be listed in the following format::
+
+	<app_service_setting_name> : <service_name or id>
+
+In the above example, ``catalog_db`` is the name of the service in your :file:`app.py` and ``test`` is the name of the service on the portal.
+
+.. tip::
+
+    Run ``tethys gen portal`` to create a blank template of this file. By default the file will be saved to ``$TETHYS_SRC/tethy_portal``.
+
+
+3. Restart Tethys Server
+==========================
+
+Restart tethys portal to effect the changes::
+
+    (tethys) $ tethys manage start
+
+.. tip::
+
+    Use the alias `tms` as a shortcut
+
+4. Configure App Settings
+=========================
+
+Set all required settings on the application settings page in the Tethys Portal admin pages (see :doc:`../../tethys_portal/admin_pages`).
+
+5. Initialize Persistent Stores
```

Comment:
Does the install script now do this? What about initializing a Geoserver? Are we going to add hooks for those too?

Reply:
I'll look into this more.. but for now I changed it to say "Set any additional required settings..."

Reply:
It does setup all of the applications settings. I don't think the scripts do anything for geoserver (yet). I would say we either leave it with my updated phrasing or turn it into a note in the previous step.  What would you prefer?  Does it seem like I am understanding your question?

Reply:
We need to add the capability to run syncstores automatically as part of the install.

Reply:
@tylorbayer regarding initializing persistent stores, if it wasn't initialized, this is how you do it manually.

Reply:
Resolved

Reply:
@tylorbayer and @msouff this section needs to be rephrased now that syncstores is being run automatically by the install command. Something like: "The `install` command will automatically run the `syncstores` command if it detects that the app has persistent stores. However, if you need to run `syncstores` manually, it can be done like so:"

Reply:
Also add "(Optional)" to the title of this section.